apps/testing-javascript/src/data/sanity-import.ts

```diff
@@ -283,16 +302,39 @@ const importCourseData = async () => {
   const uniqueLessons: {[lessonSlug: string]: LessonData} = {}
 
   for (const product of products) {
+    // they have already been parsed, no reason to re-parse them individually
+    // here
     const parsedProduct = productSchema.parse(product)
 
     const {slug: productSlug} = parsedProduct
 
+    // ignore courses with the following slugs because they are the Series
+    // duplicates. The up-to-date courses are Playlist records with the other
+    // slugs.
+    // Series.where(id: [214, 406, 223, 407, 409, 408, 246, 410]).pluck(:slug)
+    const ignoredCourseSlugs = [
+      'static-analysis-testing-javascript-applications-71c1',
+      'test-node-js-backends',
```

Comment:
I decided to hard-code this list of slugs that I pulled from `egghead-rails` rather than doing anything fancy. This ensures that only the 8 `Playlist` versions of the courses get pulled in as Workshop records.

Reply:
ah ya, this is way better. We don't need heavy reusability.